### PR TITLE
16048 Fix broken styling for detail component

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#16048

*Description of changes:*

I set the padding of the `v-textarea` to match the padding of the content inside the src/components/common/Certify.vue component (included in the screenshots). In that component, the content is wrapped with a `v-row` with `class` attributes to add padding. I did the same for the `v-textarea`, however it also works if it is wrapped in a plain old `div` with the same `class` attributes. I went with the `v-row` for consistency, however I also get if it doesn't make the most sense semantically and would be open to using a `div` instead.

I'll get UX to have a look after reviews to see if they have any preferences on the padding amount.

*Screenshot of changes:*

fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/DetailComment.vue
Comment (severinbeauvais) [inline src/components/common/DetailComment.vue:16]: Thanks for the great description of changes in the PR 👍 

Vuetify renders a v-row as a div with some classes, so you're right -- underneath, they're equivalent.

In this case, I recommend a div instead, since you're not actually using any row functionality (such as columns within it).

With a div, you may have to explicitly add some Vuetify classes that "row" automatically adds, like flexbox. Ref: https://vuetifyjs.com/en/styles/flex/

Comment (severinbeauvais) [inline src/components/common/DetailComment.vue:16]: This DetailComment component is used by various filings. You need to check all of them to ensure this extra padding doesn't mess those up.

PS Check out the [Vuetify spacing helper classes](https://vuetifyjs.com/en/styles/spacing/) -- you could use py-something here.

Comment (leodube-aot) [reply src/components/common/DetailComment.vue:16]: That makes sense, I'll switch it to a `div`. I did a quick test comparing `v-row` and `div` and didn't notice any immediate differences but I'll double check to make sure that there are no Vuetify classes for `v-row` that need to be added.

Comment (leodube-aot) [reply src/components/common/DetailComment.vue:16]: Good call, I only checked the `DetailComment` for corrections. I'll track down where else it is used and make sure the extra padding doesn't affect it. 

Also yeahI think it would look cleaner using `px-#` and `py-#` since the vertical/horizontal numbers are the same.

File: src/views/Correction.vue
Comment (severinbeauvais) [inline src/views/Correction.vue:79]: Do you need `no-gutters` here? I usually see it used for v-rows, which do something weird with left/right (negative) margins.

You might also be able to put the px/py classes right in the DetailComment element below.

**Review by severinbeauvais — APPROVED** [review]
The screenwhot and changes LGTM.

You'll need to rebase and update the app version again, sorry.

Wait for another review approval or 2 before merging.

**Comment by leodube-aot** [comment]
So turns out it does affect the spacing in the other places where `DetailComment` is used. I brought the `div` with padding to the `Correction.vue` component to wrap `DetailComment` so that it does not affect the layout of the `DetailComment` in other places. This does not quite result in the same fix (see screenshot) and I needed to add a `v-card` around the entire thing to get it to work.

_Wrapped only in div_
wrap detail comment

_Wrapped in div and in v-card_
wrap in card

File: src/views/Correction.vue
Comment (leodube-aot) [reply src/views/Correction.vue:79]: Just removed `no-gutter` and it didn't change anything. I'll add a new commit removing it.

I think the px/py classes have to be outside the DetailComment to not affect it in other places, see my comment below for more details.

Comment (leodube-aot) [reply src/views/Correction.vue:79]: I misunderstood I see what you mean by adding it right in the DetailComment. I added it and it works so I should be good to remove the div.

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

**Comment by JazzarKarim** [comment]
Btw Leo, not really sure if you used it before, you can run `/gcbrun` to generate a Temporary URL so that you can test your changes there.

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-522-zw0l23dz.web.app

**Comment by leodube-aot** [comment]
> Btw Leo, not really sure if you used it before, you can run `/gcbrun` to generate a Temporary URL so that you and other Devs can test your changes there.

Thanks for the tip! I've never used `/gcbrun` before but I'll have to start using it more often

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-522-zw0l23dz.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-522-zw0l23dz.web.app

**Comment by severinbeauvais** [comment]
> > Btw Leo, not really sure if you used it before, you can run `/******` to generate a Temporary URL so that you and other Devs can test your changes there.
> 
> Thanks for the tip! I've never used `/******` before but I'll have to start using it more often

Careful with that keyword -- just having it in a comment runs it. "That which shall not be named."

**Comment by JazzarKarim** [comment]
Looks Great!
image

I just used the temporary URL and appended the identifier to it, like this: https://business-filings-dev--pr-522-zw0l23dz.web.app/BC0870818/

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-filings-dev--pr-522-zw0l23dz.web.app

**Comment by leodube-aot** [comment]
> Wait for another review approval or 2 before merging.

I don't believe I have merge access yet so up to you on this.

Also I see what you mean about being carful with that keyword haha

**Pull request merged**
